output.tf

```diff
@@ -1,4 +1,20 @@
-#output "example" {
-#  value       = var.example
-#  description = "This is just an example of an output"
-#}
+output "service_account_name" {
+  value       = local.service_account_name
+  description = "The Service Account name"
+}
+
+output "service_account_private_key" {
```

Comment:
This may cause a sensitive output error; do we actually need it as an output?

Reply:
probably not. I can remove this

Reply:
We do have it marked as `sensitive   = true`.
Perhaps we only want to set this output if we are creating the Service account for the user?